### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by PSP92**
I added a light pink background to all the pages. Let me know if it's too much. I also styled the sign in and sign up pages to match the login page.

### CODE REVIEW

File: src/style/_loginform.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import "./variables.scss";
+.login-part{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family:$comfortaa;
+}
+h1 {
```
Comment (wtruscott) [inline src/style/_loginform.scss]: this will effect every h1 in the app it needs to be nested or more specific.

File: src/style/_loginform.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    color: $dark-green;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-family: $comfortaa;
+}
+p {
```
Comment (wtruscott) [inline src/style/_loginform.scss]: same problem here

File: src/style/_loginform.scss (hunk 1 above)
Comment (PSP92) [reply src/style/_loginform.scss]: got it ill change it

File: src/style/_signup.scss [hunk 3]
```diff
@@ -1,5 +1,20 @@
 @import "./variables.scss";
-
+.signin-page{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family:$comfortaa;
+}
+h1 {
```
Comment (wtruscott) [inline src/style/_signup.scss]: and here

File: src/style/_loginform.scss (hunk 2 above)
Comment (PSP92) [reply src/style/_loginform.scss]: got it ill change it

File: src/style/_signup.scss [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+    color: $dark-green;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-family: $comfortaa;
+}
+p {
```
Comment (wtruscott) [inline src/style/_signup.scss]: and here

File: src/style/_signup.scss [hunk 5]
```diff
@@ -26,32 +41,18 @@
...
 }
-.signin-page{
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    font-family:$comfortaa;
-}
-.signup-bttn {
-    background-color:$dark-green; 
+button{
```
Comment (wtruscott) [inline src/style/_signup.scss]: here

**Review by wtruscott — COMMENTED** [review]
So I've left comments about improperly nested css keys

File: src/style/_signup.scss (hunk 3 above)
Comment (PSP92) [reply src/style/_signup.scss]: fixed

File: src/style/_signup.scss (hunk 4 above)
Comment (PSP92) [reply src/style/_signup.scss]: fixed

File: src/style/_signup.scss (hunk 5 above)
Comment (PSP92) [reply src/style/_signup.scss]: fixed

**Review by wtruscott — APPROVED** [review]

**Pull request merged**